README.md

```diff
@@ -21,9 +21,8 @@ This will create the srk binary locally. To install to your GOPATH:
 Create a zip file containing the example workload:
 
 ```
-./srk package \
+./srk function create \
```

Comment:
I tend to think of `function create` as installing the function in the FaaS environment. E.g., compare with `create-function` on lambda ([docs here](https://docs.aws.amazon.com/cli/latest/reference/lambda/create-function.html)). So I think there should be a different command for build command for creating the zip file. The `package` command is adapted from [Maven](https://maven.apache.org/guides/getting-started/maven-in-five-minutes.html) but I'm open to alternatives.

Reply:
I was thinking of having 'function create' do both package and install (I just hadn't implemented install yet). What are the use cases for package independent of install? I suppose users may want to manually edit the package prior to install when they need to do something weird? It's easy enough to do either.